validation-test/compiler_crashers_fixed/00058-get-self-type-for-container.swift

```diff
@@ -6,6 +6,6 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
 // RUN: %target-swift-frontend %s -parse -verify
```

Comment:
compiler_crashers tests should not use -verify -- just remove the expected-errors here and the -verify flag